nifi-framework-api/src/main/java/org/apache/nifi/action/FlowActionAttributes.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.action;
+
+/**
+ * Flow Action attributes for consistent naming
+ */
+public interface FlowActionAttributes {
```

Comment:
This interface should be removed since it is just a simple wrapper right now. The `FlowActionAttribute` does not seem to provide much value, so I recommend removing it and naming the top-level `enum` as `FlowActionAttribute`.